core/storage/translation/gae_models.py

```diff
@@ -34,6 +35,159 @@
 datastore_services = models.Registry.import_datastore_services()
 
 
+# TODO(#14537): Make sure that domain object files do not import services.
```

Comment:
This is a bit confusing, I think it should say something like "This is a duplicate of the same TypedDict in xxx, it should be removed after we refactor our importing."

Reply:
Done.